modules/tribes/client/api/tribes.api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export async function join(tribeId) {
+  const { data } = await axios.post(`/api/users/memberships/${tribeId}`, { tribeId });
```

Comment:
You don't need to pass `tribeId` as a parameter here, in the angular version the parameter is just used to replace the path parameter:

```js
  function UserMembershipsService($resource) {
    return $resource('/api/users/memberships/:tribeId?', {
      tribeId: '@tribeId'
    }, {
      post: {
        method: 'POST'
      },
      delete: {
        method: 'DELETE'
      },
      get: {
        method: 'GET'
      }
    });
  }
```

The server gets the `tribeId` from params (which is path params):

https://github.com/Trustroots/trustroots/blob/303e23277de3fce2eafb32988bbe286de276dc03/modules/users/server/controllers/users.profile.server.controller.js#L904

Reply:
Careful observation. :+1: Removed body from the request and tested that it still works in fa91c259bb1bc13ed8b98c1aca7f10e085b12e21